.secrets.baseline

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "1.1.0",
+  "version": "1.2.0",
```

Comment:
This is from updating my version of `detect-secrets`.

https://artsy.slack.com/archives/C02BC3HEJ/p1656530988505479